packages/xo-web/src/common/intl/locales/fr.js

```diff
@@ -228,7 +228,7 @@ export default {
   userPage: 'Utilisateur',
 
   // Original text: "No support"
-  noSupport: 'Pas de support',
+  noSupport: 'Support professionnel manquant',
```

Comment:
« Pas de support professionnel » would be more logical in French

Reply:
Changes made